src/Router.php

```diff
@@ -17,14 +17,71 @@ public static function getInstance()
         return static::$instance;
     }
     
+    /**
+     * Loads the location of the routes file.
+     * 
+     * @return string
+     */
+    protected function getRouteFileLocation()
+    {
+        $dir = getenv('HOME') . '/.volcano-sdk';
+        
+        if (!is_dir($dir)) {
+            mkdir($dir);
+        }
+        
+        return $dir . '/routes.json';
+    }
+    
+    /**
+     * Loads Routes
+     */
     public function loadRoutes()
     {
+        $rfile = $this->getRouteFileLocation();
+        
+        if (!file_exists($rfile)) {
```

Comment:
@dsposito - What about checking the modified time on the file and re-loading it if it's older than say, 24 hours? It'd be adding a filesystem check every run, but 1 shouldn't be _too_ much overhead.